common/utils/category_resolver.py

```
@@ -0,0 +1,43 @@
+from typing import Dict
```

Ruff (0.12.2)

`typing.Dict` is deprecated, use `dict` instead

(UP035)